### DISCUSSION TITLE
Error regarding timeou?

### DISCUSSION THREAD

**Discussion by revilo100** [postalserver/postal#2902] [Help with using Postal]
We got soft fails for some emails with the message below, but I'm not sure what's the problem, with postfix I can send to it just fine and the mx record are resolving to mail._domain_.hu just fine.

**_Net::ReadTimeout with #<TCPSocket:(closed)>_**

![image](https://github.com/postalserver/postal/assets/47359361/0c0a23f8-df34-4daf-b116-bfaee613603d)

**Comment by willpower232** [comment]
Presuming this is still happening and you don't have any kind of firewall or rate limiting on that SMTP server, I wonder if this is what it looks like when Postal gives up on the MX records and goes for an A record perhaps?
